Objects/Objects/GIS/GisPolygonGeometry3d.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections.Generic;
+using Objects.Geometry;
+using Speckle.Core.Kits;
+using Speckle.Core.Models;
+
+namespace Objects.GIS;
+
+public class GisPolygonGeometry3d : Base
+{
+  public string units { get; set; }
+  public Polyline boundary { get; set; }
+  public List<Polyline> voids { get; set; }
+
+  public GisPolygonGeometry3d()
+  {
+    voids = new List<Polyline>();
+    units = Units.Meters;
```

Comment:
I see what you're doing here @KatKatKateryna,
And sorry, I think I may have given you some bad advice by saying its fine to defaulting values in constructor. 

For empty lists, I have no issue. 

But I don't think we should EVER default Units, its just too dangerous and never desirable, only opening us up to the possibility that we receive data in the wrong units.

I would rather us send an null here, and let it fail on the receiving end.

Reply:
Kat's (quite correctly) pointed out that we are doing exactly this in our mesh class, and that this is our current way to do things. 
We'll come up with a better structure as part of https://spockle.atlassian.net/browse/CNX-9352